src/sonic-framework/tests/test_utils_common.cpp

```diff
@@ -26,4 +26,21 @@ void TestUtils::wait_for_finish(swss::Select &s,
   EXPECT_EQ(sel, &finished);
 }
 
+void TestUtils::clear_tables(swss::DBConnector &db) {
```

Comment:
I think there are no callers of this, remove ?

Reply:
Removed this file.